src/pages/workspaces/workspace/notebooks/TerminalLauncher.js

```diff
@@ -101,10 +101,20 @@ const TerminalLauncher = _.flow(
       })
     } else {
       return div({ style: { padding: '2rem' } }, [
+        'Creating Stopping Starting Updating Deleting'.includes(clusterStatus) &&
         spinner({ style: { color: colors.primary(), marginRight: '0.5rem' } }),
-        (clusterStatus === 'Creating' || !cluster) ?
-          'Creating runtime environment. You can navigate away and return in 5-10 minutes.' :
-          'Starting runtime environment, this may take up to 2 minutes.'
+        Utils.cond(
```

Comment:
nit: this may be cleaner with a switchCase since all conditions are being checked against the same variable.

Reply:
+1